rllib/agents/dqn/dqn_tf_policy.py

```diff
@@ -369,8 +373,11 @@ def postprocess_nstep_and_prio(policy, batch, other_agent=None, episode=None):
 
     # Prioritize on the worker side
     if batch.count > 0 and policy.config["worker_side_prioritization"]:
+        actions = batch[SampleBatch.ACTIONS]
```

Comment:
Why do we need the "flattening" here?

I understand that RLlib requires a vector of actions, but that is a known limitation that is true for all  algorithms, isn't it?